lib/smart_answer_flows/maternity-paternity-calculator/questions/earnings_for_pay_period_adoption.govspeak.erb

```diff
@@ -7,5 +7,5 @@
 <% end %>
 
 <% content_for :error_message do %>
-  Please enter a number greater than 0
-<% end %>
+  Use numbers like 12.34
```

Comment:
Intl is build into rails right?
As these are the same, could we use Intl to DRY these out into a common error message?

Reply:
Ooh yes good point. I can abstract that.

Reply:
Also not a blocker, just a though, have approved!

Reply:
@huwd We don't use Intl in Smart Answers but it can go into a partial

Reply:
[I DRY-ed it up](https://github.com/alphagov/smart-answers/pull/4212/commits/967221d40c757912517bc7db65c30f83ca05d9e2)